src/server/services/hli/hli.test.ts

```diff
@@ -0,0 +1,74 @@
+import { StatusLineItem } from '../../../universal/types';
+import { ZorgnedAanvraagTransformed } from '../zorgned/zorgned-config-and-types';
+import { forTesting } from './hli';
+
+describe('HLI', () => {
+  test('getDisplayStatus', () => {
+    const regeling1 = {
+      productIdentificatie: 'AV-UPCC',
+      datumEindeGeldigheid: '2024-08-29',
+      datumIngangGeldigheid: '2024-08-29',
+      resultaat: 'toegewezen',
+    } as ZorgnedAanvraagTransformed;
+
+    const statusLineItems1 = [] as StatusLineItem[];
+
+    expect(forTesting.getDisplayStatus(regeling1, statusLineItems1)).toBe(
+      'Afgewezen'
```

Comment:
op regel 11 staat `toegewezen` hoe kan dit dan `Afgewezen` worden?

Reply:
Ik heb mijn eigen antwoord gevonden in de `isCursusNietVoldaan`, dus klopt nvm :)